pkg/resource/training_job/hooks.go

```diff
@@ -31,12 +32,21 @@ var (
 		svcsdk.RuleEvaluationStatusInProgress,
 		svcsdk.RuleEvaluationStatusStopping,
 	}
+	WarmPoolModifyingStatuses = []string{
+		svcsdk.WarmPoolResourceStatusAvailable,
+		svcsdk.WarmPoolResourceStatusInUse,
+	}
 	resourceName = GroupKind.Kind
 
 	requeueWaitWhileDeleting = ackrequeue.NeededAfter(
 		errors.New(resourceName+" is Stopping."),
 		ackrequeue.DefaultRequeueAfterDuration,
 	)
+
+	requeueWaitWhileWarmPoolInUse = ackrequeue.NeededAfter(
+		errors.New("Warm Pool Cluster is still active."),
```

Comment:
provisioned infrastructure is retained, using words from docs: https://docs.aws.amazon.com/sagemaker/latest/dg/train-warm-pools.html

Reply:
Yea that is better wording